hooks/usePoll.ts

```
@@ -0,0 +1,91 @@
+
+        const res = await fetch(`/poll?${params.toString()}`);
+        if (!res.ok) throw new Error("Failed to fetch polls");
+
+        const data = await res.json();
+        return data.polls;
+      },
+    });
+  };
+
+  const getPollDetails = (id: number) => {
+    return useQuery({
```

ESLint

[error] React Hook "useQuery" is called in function "getPollDetails" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)